float32 confidence
uint8 team_color
uint8 YELLOW=0
uint8 BLUE=1
uint8 robot_id
geometry_msgs/Pose robot_pose
float32 pixel_x
float32 pixel_y
